model-archiver/model_archiver/tests/unit_tests/test_model_packaging_utils.py

```diff
@@ -75,6 +75,13 @@ def test_export_file_is_none_tar(self, patches):
             patches.path_exists.assert_called_once_with("/Users/dummyUser/some-model")
             assert ret_val == "/Users/dummyUser"
 
+    # noinspection PyClassHasNoInit
+    class TestArchiveTypes:
+        def test_archive_types(self):
+            from model_archiver.model_packaging_utils import archiving_options as ar_opts
+            assert ar_opts.get("tgz") == ".tar.gz"
+            assert ar_opts.get("no-archive") == ""
+            assert ar_opts.get("default") == ".mar"
```

Comment:
Can you also add another test here to ensure the keys are in the set : ```tgz, no-archive, default```, that way if we add a new format in future, we'll make sure we update this test too for the exhaustive checks. :)

Reply:
Ahh. Sure.